src/slicer_connection_interface.py

```diff
@@ -0,0 +1,110 @@
+import asyncio
+import pyigtl
+import time as t
+import logging
+
+from src.common.api.mct_request_series import MCTRequestSeries
+from src.common.structures.component_role_label import COMPONENT_ROLE_LABEL_DETECTOR, COMPONENT_ROLE_LABEL_POSE_SOLVER
+from src.controller.mct_controller import MCTController
+from src.controller.structures.mct_component_address import MCTComponentAddress
+from ipaddress import IPv4Address
+
+from src.pose_solver.api.add_target_marker_request import AddTargetMarkerRequest
+from src.pose_solver.api.set_reference_marker_request import SetReferenceMarkerRequest
+
+from src.controller.structures.connection import Connection
+
+pose_solver_label = "p"
+detector_labels_and_IPs = [
+    ("d101",IPv4Address('192.168.0.101')),
+    ("d102",IPv4Address('192.168.0.102'))]
```

Comment:
These look specific to our in-lab setup

Leave it here for now, but please add a comment that this needs to be updated/removed. Eventually we will want to send to the controller a configuration file, and that will contain the initialization instructions.